recipes/vcglib/all/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Great job! The devel branch provides a cmake file, but yours looks concrete and similar.
https://github.com/cnr-isti-vclab/vcglib/blob/devel/CMakeLists.txt